src/pages/authorization/scopes/index.mdx

```diff
@@ -77,7 +77,7 @@ Scopes define access to certain resources. Your app will ask users for access to
 
 ## Properties scopes
 
-<Scopes scopes="properties--my:ro,properties--my:rw,properties--all:ro,properties--configuration:rw" />
+<Scopes scopes="properties--my:ro,properties--my:rw,properties--all:ro,properties.configuration:rw,properties.license.value--my.rw,properties.license.value--all.rw,properties.group.value--my.rw,properties.group.value--all.rw" />
```

Comment:
I think we can already remove old scopes `properties--my:ro` etc, if they are not used anymore in recent stable (v3.4+) version. Wdyt @ale-kacperczyk ?

Reply:
Yes, that's the approach we've adopted